### ISSUE TITLE
error creating env on Windows: dlc-windowsGPU.yaml: ResolvePackageNotFound:   - tensorflow-gpu==1.13.1 

### ISSUE THREAD

**Issue by effiek9999** [DeepLabCut/DeepLabCut#235]
**Operating system and DeepLabCut version:**

Windows 10, latest DeepLabCut, following these instructions: https://github.com/AlexEMG/DeepLabCut/blob/master/docs/installation.md
Anaconda env
GPU: GeoForce GTX 1080Ti
pip install tensorflow-gpu worked independent;y from DeepLabCut and recognizes the GPU

**Describe the problem:**
Error setting up environment for windows by using command 

> conda env create -f dlc-windowsGPU.yaml

With 'tensorflow-gpu' changed to 1.13.1:
tensorflow-gpu==1.13.1

Though when I do a 'conda list', I see this: (attached screenshot)
tensorflow-gpu            1.13.1                   pypi_0    pypi

**Steps to reproduce the behavior:**
1. Do pip install tensorflow-gpu (see attachment)
2. Run 'conda env create -f dlc-windowsGPU.yaml' from c:\Users\Usertytell_admin\Anaconda3\envs which contains the file dlc-windowsGPU.yaml
3. Error results: 
Collecting package metadata: done
Solving environment: failed

ResolvePackageNotFound:
  - tensorflow-gpu==1.13

**More details are in these screenshots:**
![Screenshot (3)](https://user-images.githubusercontent.com/3596849/54058066-c3720080-41c2-11e9-8fe9-2bf8d8f9bf31.png)

![Screenshot (2)_LI](https://user-images.githubusercontent.com/3596849/54058076-cb31a500-41c2-11e9-94a7-2439f2d55e25.jpg)

**What happened earlier:**

![1](https://user-images.githubusercontent.com/3596849/54058458-dd601300-41c3-11e9-947c-f63a635daded.png)

**Did a successful pip install tensorflow-gpu:**

![2](https://user-images.githubusercontent.com/3596849/54058459-dd601300-41c3-11e9-91e7-a3b9985d2c29.png)
![3](https://user-images.githubusercontent.com/3596849/54058460-dd601300-41c3-11e9-95fb-64d3d7964cc6.png)

**Comment by MMathisLab** [comment]
this has nothing to do with DLC... please use the conda as provided, or please open the issue on TF.

**Comment by effiek9999** [comment]
Do you mind explaining "please use the conda as provided"? I cannot find comprehensive instructions for this in any particular place, it makes this installation more painful than it needs to be I think. I have been following all instructions I can find. Thanks.

**Comment by MMathisLab** [comment]
I mean, why did you change the TF version inside the conda file? The installation is simply download the code, in terminal (when you are inside the conda.. file) run the command you already did. I’m not sure what the issue is.

**Comment by effiek9999** [comment]
Oh you mean I should not change the tensorflow-gpu version in the yaml file? Ok, it has been a bit of a maze trying to figure out what versions of what work together, so with the Geoforce GTX 1080ti I have CUDA 10 and TensorFlow 1.13.1, so I assumed I would have to change the TensorFlow version in the yaml file. I will try it without changing the version, it just doesn't make sense to me why conda list shows tensorflow1-gpu as 1.13.1 and why using that in the yaml file wouldn't work.

**Comment by effiek9999** [comment]
Just as an update, I finally managed to run `conda env create -f dlc-windowsGPU.yaml` with the yaml file unchanged, but many errors followed which I fixed by doing a 'conda install <whatever pip could not install>' one-by-one as the errors came up. Now that I started using DeepLabCut in this environment I am still not sure if the correct thing to do` deeplabcut.train_network` is to run `activate  dlc-windowsGPU` and then run `import deeplabcut` and then ` deeplabcut.train_network` and if that is sufficient to ensure that DeepLabCut and Tensorflow are using the GPU. I posted a topic in Image.sc asking the same question, I am sure it is my lack of experience that is making this unclear to me. Thanks.
